### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#179] [Esercizi - Bertini]
![Screenshot 2023-11-13 194621](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/7cffe78e-a31c-4f6d-b5be-0d420f7c9197)

**Comment by Elia-Belli** [comment]
Per l'indipendenza $\mathbb{P}(X_1=x_1,X_2=x_2)=\mathbb{P}(X_1=x_1)\mathbb{P}(X_2=x_2)$
Per l'uniformità $\mathbb{P}(X_1=x_1)=a,\forall x_1\in Im(X_1)$ e $\mathbb{P}(X_2=x_2)=b,\forall x_2\in Im(X_2)$ 
E siccome sono uniformi sullo stesso intervallo $a=b$

- **(1)** Siccome le v.a. sono indipendenti sappiamo che:
$$\mathbb{P}(X_1+X_2=z)=\sum^{min(n,z)}\_{x_1=1}\mathbb{P}(X_1=x_1)\mathbb{P}(X_2=z-x_2)=\sum^{min(n,z)}\_{x_1=1}a\cdot b=min(n,z)\cdot a^2\ \ \ \ \ \ \ \ z=2,...,2n$$
- **(2)** Per linearità del valore atteso:
$$\mathbb{E}(X_1+X_2)=\mathbb{E}(X_1)+\mathbb{E}(X_2)=\sum^n_{x_1=1}x_1\cdot a+\sum^n_{x_2=1}x_2\cdot b=2a\cdot \frac{n\cdot (n+1)}{2}=an\cdot (n-1)$$
- **(3)** Siccome le v.a. sono indipendenti $\mathbb{V}(X_1+X_2)=\mathbb{V}(X_1)+\mathbb{V}(X_2)$:
$$\mathbb{V}(X_1)+\mathbb{V}(X_2)=[\mathbb{E}(X_1^2)-\mathbb{E}(X_1)^2]+[\mathbb{E}(X_2^2)-\mathbb{E}(X_2)^2]$$

Inoltre $\mathbb{E}(X_1)=\mathbb{E}(X_2)=a\cdot \frac{n\cdot (n+1)}{2}$

$$\mathbb{V}(X_1+X_2)=2\cdot [\mathbb{E}(X_1^2)-\mathbb{E}(X_1)^2]=2\cdot \bigg[ (\sum^n_{x=1}x^2\cdot a) - (a\cdot \frac{n\cdot (n+1)}{2})^2\bigg]=$$

$$=a\cdot \bigg[\frac{n(n+1)(2n+1)}{3}-(a\cdot \frac{n^2\cdot (n+1)^2}{2})\bigg]$$

**Reply by CasuFrost** [reply]
Solo una piccola precisazione, nel punto 1 la sommatoria non dovrebbe arrivare fino al minimo tra $n$ e $z$, ma fino al minimo tra 
$n$ e $z-1$, in quanto la probabilità che la somma faccia $z$, se uno dei due è $z$ è nulla in quanto non possono assumere valore 0, essendo uniformi in $\{1,2\dots,n}$ .

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/9bdb8576-b950-47a1-8656-5f2745a1c31e)

**Comment by cornflexxx** [comment]
1)

Se le variabili aleatorie prese in considerazione sono uniformi questo significa che  presi $x_1,x_2 \in [1,n]$ , $\mathbb{P}(X_1=x_1)=\mathbb{P}(X_2=x_2)=1/n$
Allora la distribuzione della loro somma si calcola nel seguente modo:

$$\mathbb{P}(X_1+X_2 = z)=\sum_{x_1=1}^{n}\mathbb{P}(X_1=x_1)\mathbb{P}(X_2=z-x_1)=\frac{z}{n^2}$$

Il supporto della distribuzione passa da $[1,n]$ a $[2,2n]$

2)

$$\mathbb{E}(X_1+X_2)=\mathbb{E}(X_1)+\mathbb{E}(X_2) = 2\cdot\frac{n+1}{2} = n+1$$
